src/Features/Core/Portable/AddImport/CodeActions/AssemblyReferenceCodeAction.cs

```diff
@@ -7,22 +7,21 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.SymbolSearch;
 using Microsoft.CodeAnalysis.Tags;
+using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CodeFixes.AddImport
 {
     internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSyntax>
     {
-        private partial class AssemblyReference
-        {
             private class AssemblyReferenceCodeAction : CodeAction
             {
-                private readonly AssemblyReference _reference;
```

Comment:
"Reference"s held back onto the provider, allowing these code actions to perform logic based on the provider later on down the line.  I want them to only hold onto data so they can be OOPified.